eureka/src/main/java/com/linecorp/armeria/client/eureka/EurekaEndpointGroup.java

```diff
@@ -194,36 +195,38 @@ private void fetchRegistry() {
                 ctx = captor.get();
             }
             final EventLoop eventLoop = ctx.eventLoop().withoutContext();
-            response.aggregateWithPooledObjects(eventLoop, ctx.alloc()).handle((aggregatedRes, cause) -> {
-                if (closed) {
-                    return null;
-                }
-                if (cause != null) {
-                    logger.warn("Unexpected exception while fetching the registry from: {}." +
-                                " (requestHeaders: {})", webClient.uri(), requestHeaders, cause);
-                } else {
-                    try (HttpData content = aggregatedRes.content()) {
-                        final HttpStatus status = aggregatedRes.status();
-                        if (!status.isSuccess()) {
-                            logger.warn("Unexpected response from: {}. (status: {}, content: {}, " +
-                                        "requestHeaders: {})", webClient.uri(), status,
-                                        aggregatedRes.contentUtf8(), requestHeaders);
+            response.aggregate(AggregationOptions.usePooledObjects(ctx.alloc(), eventLoop))
+                    .handle((aggregatedRes, cause) -> {
+                        if (closed) {
+                            return null;
+                        }
```

Comment:
Not related to this PR, I found a possible buffer leak by change.
Should we close `aggregatedRes.content()` if `close` is true?

Reply:
Oops thanks for catching it. Fixed. 😉